src/components/NetworkSettings/NetworkStatusBar.js

```diff
@@ -0,0 +1,22 @@
+import { useSelector } from "react-redux";
+import { AlertUI } from "@styles/themes";
+import { ToplineBar } from "@components/ToplineBar";
+import { eq } from "lodash";
+import { PRE_SETTINGS_MAINNET, PRE_SETTINGS_TESTNET } from "../../constants";
+
+export const NetworkStatusBar = () => {
+  const networkSettings = useSelector((state) => state.networkSettings);
+  if (eq(networkSettings , PRE_SETTINGS_MAINNET)
+    || eq(networkSettings, PRE_SETTINGS_TESTNET)) {
+    return null;
+  }
+
+  const style = {
+    backgroundColor: AlertUI.primaryColor,
+    color: AlertUI.dark40Color,
+  };
+  const text = 'n='+networkSettings.network+' • s='+networkSettings.stage;
```

Comment:
I don't think it's clear for the user this text. I prefer having only "Custom network: X"

What do you think?

Reply:
For sure "custom network: x" is more clear. However, "stage" can be important to have in sight also. I will adopt your suggestion, but let's keep it in mind, the "stage" info.

Reply:
- Change custom network text: d2140bc72578f0d7d31cce8a91c2e342af5859fb